source/UncontrolledActivated.tsx

```diff
@@ -10,24 +17,39 @@ import {
 } from './helpers'
 import './styles.css'
 
-const UncontrolledActivated = ({
+interface Props {
+  children: React.ReactNode
+  closeText?: string
+  onUnload: () => void
+  onLoad: () => void
+  overlayBgColorEnd?: string
+  overlayBgColorStart?: string
+  parentRef?: RefObject<HTMLElement>
```

Comment:
I don't think this needs a `?`, for the parent will always send this as `{ current }` where the value of current may or may not be there, but the object is always consistent